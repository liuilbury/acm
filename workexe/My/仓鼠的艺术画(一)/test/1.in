59 12 12
47 6
52 12
28 5
37 12
19 1
3 2
18 11
4 1
18 2
53 11
5 10
37 7

