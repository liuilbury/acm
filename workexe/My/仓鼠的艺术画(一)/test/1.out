* * * * * * * * * * * *
* 6 * * * * * * * * * *
8 6 6 * * * * * * * * *
8 8 * * * * * * * 11 * *
8 * * * * * * * 11 11 11 *
* * * * * * * * * 11 * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* 9 * * * * * * * * 7 *
9 9 9 * * * * * * 7 7 7
5 9 * * * * * * * * 7 *
5 * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * 3 * * * * * * *
* * * 3 3 3 * * * * * *
* * * * 3 * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * 12 * * * * 4
* * * * * 12 12 12 * * 4 4
* * * * * * 12 * * * * 4
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * 1 * * * * * *
* * * * 1 1 1 * * * * *
* * * * * 1 * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * 2
* * * * * * * * * * 10 2
* * * * * * * * * 10 10 10
* * * * * * * * * * 10 *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
* * * * * * * * * * * *
